source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.h

```diff
@@ -10,20 +10,21 @@
 #import "ChoiceSetInput.h"
 #import "ACRIBaseInputHandler.h"
 #import "HostConfig.h"
+#import "ACOHostConfig.h"
+#import "ACOHostConfigPrivate.h"
 
-extern NSString *checkedCheckboxReuseID;
-extern NSString *uncheckedCheckboxReuseID;
-extern NSString *checkedRadioButtonReuseID;
-extern NSString *uncheckedRadioButtonReuseID;
+extern NSString *choiceSetCellReuseID;
 
 @interface ACRChoiceSetCell : UITableViewCell
 @end
 
 @interface ACRChoiceSetViewDataSource:NSObject<UITableViewDataSource, UITableViewDelegate, ACRIBaseInputHandler>
 @property NSString *id;
+@property ACOHostConfig *hostConfig;
 @property BOOL isMultiChoicesAllowed;
 
-- (instancetype)initWithInputChoiceSet:(std::shared_ptr<AdaptiveCards::ChoiceSetInput> const&)choiceSet;
+- (instancetype)initWithInputChoiceSet:(std::shared_ptr<AdaptiveCards::ChoiceSetInput> const&)choiceSet
+                         andHostConfig:(ACOHostConfig *)hostConfig;
```

Comment:
please remove "and", you can leave it as HostConfig

Reply:
ok